CRM/Case/Form/Activity/OpenCase.php

```diff
@@ -9,12 +9,6 @@
  +--------------------------------------------------------------------+
  */
 
-/**
```

Comment:
@totten are we able to remove these blocks now? I had in mind we still needed them for 'some reason'

Reply:
posted comment in wrong spot - retyping here - this block is a duplicate of the block above, so seems ok to remove.

Reply:
Is it? I mean most files seem to have both the licensing block & one with a more docblock link - ie https://github.com/civicrm/civicrm-core/blob/9c1bc317913553a956288e52649f1c372dc7deb0/CRM/Utils/Array.php#L16

I'm not sure that's a duplication

Reply:
I see what you mean. It seems some files have none of it, some have both, and some have just the docblock with the `@package` line but not the `@copyright`.

Reply:
Yep!

Reply:
@eileenmcnaughton  IMHO, the standard block higher up is the more important block.

AFAIK, these blocks with `@package` and `@copyright` were originally/primarily targeted at `phpDocumentor`... which had been used for a bit circa Civi 2.x or 3.x, but we haven't been running it since then. It's possible that `phpunit` may have respected `@package` at some point (eg pre-php53?), but nowadays `phpunit` seems to organize by the php53 namespace.

So, as near as I can tell, it's OK to get rid of these annotations. Unless y'all think we should angle to run phpDocumentor again, I think it's probably just as well to remove the redundant `@package` and `@copyright`.

NB there are a handful of oddball files with an atypical copyright/license assertion (e.g. using Academic Free License or somesuch that's permitted by the exception). If the annotation is an oddball, then I wouldn't recommend any pro-forma deletions on that.

Reply:
OK then - I'll merge - I think it would be better to have a preferred version agreed though not 'either/or'